EU4toV2/Data_Files/configurables/culture_definitions.txt

```diff
@@ -490,6 +490,52 @@ pre_indo_european = {
 			Punicid Orisid Amidian Nisunidian Tiresidian Budidian Gargoridian Culcidian Indicidian Iscerbelidian Karidian Korbidian Lagunidian Mandonidian Magavarician Uxentican Umarilid Turenidian Urceticidian
 		}
 	}
+	etrurian = {
```

Comment:
The Etrurian name pool should use Etrurian rather than Latin names, same goes for Rhaetian

Reply:
I think grabbed the Etruscan names from EU IV, but that sounds good, when I get the chance I'll go ahead and give them Etruscan names from Imperator